src/parse-stream.js

```diff
@@ -369,7 +369,7 @@ export default class ParseStream extends Stream {
           tagType: 'program-date-time'
         };
         if (match[1]) {
-          event.dateTimeString = match[1];
+          event.dateTimeString = (" " + match[1]).substring(1);
```

Comment:
```suggestion
          event.dateTimeString = (' ' + match[1]).substring(1);
```
Just realized this fails on lint due to the quote rule. Can we change this to a single quote?